README/ReleaseNotes/v630/index.md

```diff
@@ -193,13 +193,19 @@ Note that not all of the C++ types supported in TTree are currently supported in
 
 - Many bug fixes and performance improvements
 
-Please, report any issues regarding the abovementioned features should you encounter them.
+Please, report any issues regarding the above mentioned features should you encounter them.
 RNTuple is still experimental and is scheduled to become production grade by end of 2024.
 Thus, we appreciate feedback and suggestions for improvement.
 
 ## Histogram Libraries
 
-The new class `TScatter` has been added. More info to come!
+2D Scatter plots are a very popular way to represent scientific data. Many scientific plotting
+packages have this functionality. For many years ROOT itself as offered this kind of
+visualization dedicated drawing options for TGraph or TTree. But there was no simple way
+to produced 4D scatter plots from data stored in simple vectors. To fulfil these requirements
+the new class, TScatter, has been implemented. It is able to draw a four variables scatter
+plot on a single plot. A [detailed description](https://root.cern/blog/new-class-tscatter/)
```

Comment:
Maybe add a plot example in here? I really liked when you had pictures in the release notes in the past, it's good to have more than text sometimes!